keyboards/yunzii/common/smart_ble.c

```diff
@@ -0,0 +1,104 @@
+// Copyright 2024 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "uart.h"
+#include "ch.h"
+#include "hal.h"
+#include "host.h"
+#include "host_driver.h"
+#include "report.h"
+#include "smart_ble.h"
```

Comment:
Without this, the memcpy commands are failing. 
```suggestion
#include <string.h>
#include "smart_ble.h"
```